pkg/util/netutil/net.go

```diff
@@ -159,3 +160,21 @@ func FatalIfUnexpected(err error) {
 		log.Fatal(context.TODO(), err)
 	}
 }
+
+// InitialHeartBeatFailedError indicates that while attempting a GRPC
+// connection to a node, we aren't successful and have never seen a
+// heartbeat over that connection before.
+type InitialHeartBeatFailedError struct {
```

Comment:
nit: "Heartbeat" is one camel-cased word everywhere else in our codebase. Let's keep it the same here.